app/services/events/EventsParsingService.scala

```diff
@@ -97,4 +69,46 @@ trait EventsParsingService {
       }
     }
   }
+
+  lazy val df = DateTimeFormat.forPattern("HH:mm")
+
+  private def event(items: String): Event = {
+    event(items.split(", ?", -1))
+  }
+
+  private def event(items: Array[String]): Event = {
```

Comment:
Following my comment above, this function will take a `String` and return an `Event` so perhaps the name of the function should reflect that somehow.